// containers/marker/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import R from 'ramda'
import {Link} from 'react-router'

import {getMarkerById} from 'selectors'
import {fetchMarkerById, addMarkerToBasket} from 'actions'
import BasketCart from 'components/basketCart'

class Marker extends Component {
  componentDidMount () {
    this.props.fetchMarkerById(this.props.params.id)
  }

  renderFields () {
    const {marker} = this.props
    const columnFields = R.compose(
      R.toPairs,
      R.pick([
        'оплата',
        'доставка',
          'instagram',
          'контакты'
      ])
    )(marker)

    return columnFields.map(([key, value]) => (
      <div className='column' key={key}>
        <div className='ab-details-title'>
          <p>{key}</p>
        </div>
        <div className='ab-details-info'>
          {value}
        </div>
      </div>
    ))
  }

  renderContent () {
    const {marker} = this.props

    return (
      <div className='thumbnail'>
        <div className='row'>
          <div className='col-md-6'>
            <img
              className='img-thumbnail'
              src={marker.image}
              alt={marker.name}
            />
          </div>
          <div className='col-md-6'>
            {this.renderFields()}
          </div>
        </div>
        <div className='caption-full'>
          <h4 className='pull-right'>{marker.price} грн</h4>
          <h4>{marker.name}</h4>
          <p>{marker.description}</p>
        </div>
      </div>
    )
  }

  renderSidebar () {
    const {marker, addMarkerToBasket} = this.props
    return (
      <div>
        <p className='lead'>ПОКУПКИ</p>
        <BasketCart />
        <div className='form-group'>
          <h1>{marker.name}</h1>
          <h2>{marker.price} грн</h2>
        </div>
        <button
          type='button'
          className='btn btn-success btn-block'
          onClick={() => addMarkerToBasket(marker.id)}
        >
          Добавить в корзину
        </button>
          <Link to='/' className='btn btn-info btn-block'>Продолжить покупки</Link>
      </div>
    )
  }

  render () {
    const {marker} = this.props
    return (
      <div className='view-container'>
        <div className='container'>
          <div className='row'>
            <div className='col-md-9'>
              {marker && this.renderContent()}
            </div>
            <div className='col-md-3'>
              {marker && this.renderSidebar()}
            </div>
          </div>
        </div>
      </div>
    )
  }
}

const mapStateToProps = state => {
  return {
    marker: getMarkerById(state, state.markerPage.id)
  }
}

const mapDispatchToProps = {
  fetchMarkerById,
  addMarkerToBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(Marker)
