// api/mockMarkers.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "Маркеры 'поштучно'",
    description: "Палитра скетч-маркеров 'поштучно'. Маркеры #touchnew. Можно собрать свой любимый наборчик! Листайте фото - номера поближе. Маркеры двусторонние, острое и скошенное перо.",
    price: 30,
    image: "/uploads/1.1.jpg",
      доставка: "Отправка по Украине Новой почтой.",
      оплата: "Предоплата или наложенный платеж.",
      instagram: "@markeri_marso",
      контакты: "+380978671986"
  },
  {
    id: "2",
    categoryId: "2",
    name: "Наборы 10 штук",
    description: "Базовый набор скетч-маркеров #touchnew для начинающего иллюстратора 10 шт. Двухсторонние маркеры - скошенное и острое перо.",
    price: 300,
    image: "/uploads/10.jpg",
      доставка: "Отправка по Украине Новой почтой.",
      оплата: "Предоплата или наложенный платеж.",
      instagram: "@markeri_marso",
      контакты: "+380978671986"
  },
  {
    id: "3",
    categoryId: "3",
    name: "Наборы 20 штук",
    description: "20 маркеров в наборе. Маркеры двусторонние - острое перо и скошенное.",
    price: 600,
    image: "/uploads/20.JPG",
      доставка: "Отправка по Украине Новой почтой.",
      оплата: "Предоплата или наложенный платеж.",
      instagram: "@markeri_marso",
      контакты: "+380978671986"
  },
  {
    id: "4",
    categoryId: "4",
    name: "Наборы 30 штук",
    description: "Набор 30 штук скетч-маркеров",
    price: 900,
    image: "/uploads/30.jpg",
      доставка: "Отправка по Украине Новой почтой.",
      оплата: "Предоплата или наложенный платеж.",
      instagram: "@markeri_marso",
      контакты: "+380978671986"
  },
  {
    id: "5",
    categoryId: "5",
    name: "Наборы 40 штук",
    description: "А у меня для вас собрался роскошный набор скетчмаркеров из 40 штук!",
    price: 1200,
    image: "/uploads/40.jpg",
      доставка: "Отправка по Украине Новой почтой.",
      оплата: "Предоплата или наложенный платеж.",
      instagram: "@markeri_marso",
      контакты: "+380978671986"
  },
  {
    id: "6",
    categoryId: "6",
    name: "Наборы 60 штук",
    description: "Отличный, красочный набор скетчмаркеров 60 штук",
    price: 1800,
    image: "/uploads/60.jpg",
      доставка: "Отправка по Украине Новой почтой.",
      оплата: "Предоплата или наложенный платеж.",
      instagram: "@markeri_marso",
      контакты: "+380978671986"
  },
    {
        id: "10",
        categoryId: "6",
        name: "Наборы 60 штук",
        description: "Очень классные маркеры на водной основе! Двусторонние, перо-кисть твердо-мягкое и тоненький наконечник-лайнер. Картонный тубус-короб и пластиковый тубус для хранения, удобный в использовании. Цвета яркие, сочные. Не имеют запаха. Отлично работают на акварельной бумаге! ",
        price: 1400,
        image: "/uploads/60а.jpg",
        доставка: "Отправка по Украине Новой почтой.",
        оплата: "Предоплата или наложенный платеж.",
        instagram: "@markeri_marso",
        контакты: "+380978671986"
    },
    {
        id: "7",
        categoryId: "7",
        name: "Наборы 80 штук",
        description: "Набор скетч-маркеров 80 штук #touchnew. Маркеры в сумочке, двусторонние, скошеное и острое перья!",
        price: 2000,
        image: "/uploads/80.jpg",
        доставка: "Отправка по Украине Новой почтой.",
        оплата: "Предоплата или наложенный платеж.",
        instagram: "@markeri_marso",
        контакты: "+380978671986"
    },
    {
        id: "8",
        categoryId: "8",
        name: "Наборы 168 штук",
        description: "Та-даааам! Ну кто о таком наборе не мечтает?! 168 оттенков счастья! Очень свежие чернила, пишут абсолютно все, есть блендер, 2 линера, 2 белые ручки. Наборы в сумочках. Отличный чемоданчик интроверта, чтобы скрыться на выходные от суеты и творить. Доставка по Украине в подарок.",
        price: 3700,
        image: "/uploads/168.jpg",
        доставка: "Отправка по Украине Новой почтой в подарок.",
        оплата: "Предоплата или наложенный платеж.",
        instagram: "@markeri_marso",
        контакты: "+380978671986"
    },
    {
        id: "9",
        categoryId: "9",
        name: "Скетчбуки",
        description: "Скетчбуки специально для маркеров. 30 листов. Размеры 21х14 см. Плотность150/m²",
        price: 150,
        image: "/uploads/book.jpg",
        доставка: "Отправка по Украине Новой почтой.",
        оплата: "Предоплата или наложенный платеж.",
        instagram: "@markeri_marso",
        контакты: "+380978671986"
    }
]
